src/scripts/main.js

```diff
@@ -1,3 +1,96 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  // максимальна кіслькість рядків і стовців
+  const maxRow = 10;
+  const maxColumn = 10;
+  const minRow = 2;
+  const MinColumn = 2;
+
+  const div = document.querySelector('.container');
+  const appendRow = div.querySelector('.append-row ');
```

Comment:
There is an extra space in the query selector string `'.append-row '`. It should be `'.append-row'` to ensure the element is correctly selected.